canvasapis.py

```diff
@@ -0,0 +1,150 @@
+import logging
+import requests
+import urllib
+
+
+class GroupsForSections:
+    """
+    This class makes the api calls to create groups for sections. TODO: this class will be taken out into a separate
+    module in a different repo
+    """
+    HTTP_METHOD_POST = 'Post'
+    HTTP_METHOD_GET = 'Get'
+    AUTHORIZATION = 'Authorization'
+    BEARER = 'Bearer '
+    MIME_TYPE_JSON = 'application/json'
+    CONTENT_TYPE = 'Content-type'
+
+    def __init__(self, canvas_token, canvas_url):
+        self.canvas_token = canvas_token
+        self.canvas_url = canvas_url
+
+    def create_group_category(self, group_category_name, course_id):
+        """
+        Creates a group category with a given name for a particular course, In case of exceptions it throws it to
+        calling method
+
+        :param String group_category_name:
```

Comment:
Python documentation doesn't combine the argument (param) names and types.  They go in different fields.  For example:

```python
    def function1(self, arg1, arg2, arg3):
        """Performs the calculation: arg1 / arg2 + arg3

        :param arg1: the first value
        :param arg2: the second value
        :param arg3: the third value
        :type arg1: int, float,...
        :type arg2: int, float,...
        :type arg3: int, float,...
        :returns: result of the calculation
        :rtype: int, float
        """
```

Which is paraphrased from:  http://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html#template-py-source-file